src/hooks/useBadgeCount.ts

```diff
@@ -0,0 +1,10 @@
+import { useInitialisedDeskproAppClient } from "@deskpro/app-sdk";
+import type { Maybe, DeviceType } from "../types";
+
+const useBadgeCount = (items: Maybe<DeviceType[]>) => {
```

Comment:
Type needs updating to `any[]` as it doesn't care about what is contained in the array. Or maybe use `count: number` instead. Eitherway, it doesn't need `Maybe` as it is and must always be called with an `array`, there is no reason to call this without an entry.

Reply:
There are two points here:
1. We have a rule - not to use the type any.
2. Based on the application context, only `DeviceType[]` can be used here.
3. We have already discussed this hook. And I implemented it as we agreed

Reply:
1. To avoid `any`, use `count: number`.
2. We should follow separation of concern, the `useBadgeCount` doesn't need to know anything about devices or `DeviceType`. It's coupling things that don't need need to be coupled.
3. We discussed in Slack but you implemented something else that was not agreed.

Reply:
This hook calculates the number of assets. So everything is correct here with concern